api/__init__.py

```diff
@@ -0,0 +1,54 @@
+import base64
+
+from flask import Flask, request
+import numpy as np
+
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route('/')
+    def routes():
+        return {
+            "spectrum": "/spectrum",
+            "mean": "/mean",
+            "median": "/median",
+        }
+
+    @app.route('/spectrum', methods=['POST', 'GET'])
+    def spectrum():
+        if request.method == 'GET':
+            return {'params': ['sampling_rate', 'units', 'dtype']}
+
+        req = request.get_json()
+        sr = float(req['sampling_rate'])
```

Comment:
An exception could be thrown casting a non-float value.  A few questions here:

- [ ] Would that exception crash the server?
- [ ] Is there a graceful way to handle that use case.
- [ ] Add non-number sampling_rate to the test scripts.

Reply:
#4